pyccel/parser/semantic.py

```diff
@@ -6324,3 +6324,50 @@ def _build_ListInsert(self, expr, args):
                 and not isinstance(new_elem, (PythonList, PythonSet, PythonTuple, NumpyNewArray)):
             self._indicate_pointer_target(list_obj, new_elem, expr)
         return semantic_node
+
+    def _build_SetDifference(self, expr, function_call_args):
+        """
+        Method to visit a SetDifference node.
+
+        The purpose of this `_build` method is to construct multiple nodes to represent
+        the single DottedName node representing the call to SetDifference. It
+        replaces the call with a call to copy followed by multiple calls to
+        SetDifferenceUpdate.
+
+        Parameters
+        ----------
+        expr : DottedName
+            The syntactic DottedName node that represents the call to `.intersection()`.
```

Comment:
You mean `.difference()`...

Reply:
Fixed [2260bc9](https://github.com/pyccel/pyccel/pull/2377/commits/2260bc9a5030edf3b5a195b51cc39e09831bc093)